webapp/src/ts/modules/about/about.component.html

```diff
@@ -31,91 +33,132 @@ <h3 (click)="secretDoor()">{{'about' | translate}}</h3>
                   <span *ngSwitchCase="'browser'">{{'login.unsupported_browser.outdated_browser' | translate}}</span>
                 </ng-container>
               </p>
-            </dd>
+            </div>
           </ng-template>
-        </span>
-        <span *ngIf="androidDeviceInfo?.app?.version">
-          <dt>{{'android_app.version.title' | translate}}</dt>
-          <dd>{{androidDeviceInfo.app.version}}</dd>
-        </span>
-        <span *ngIf="androidDeviceInfo?.app?.packageName">
-          <dt>{{'android_app.package_name.title' | translate}}</dt>
-          <dd>{{androidDeviceInfo.app.packageName}}</dd>
-        </span>
-        <span *ngIf="androidDeviceInfo?.app?.versionCode">
-          <dt>{{'android_app.version_code.title' | translate}}</dt>
-          <dd>{{androidDeviceInfo.app.versionCode}}</dd>
-        </span>
-        <span *ngIf="androidDeviceInfo?.software?.androidVersion">
-          <dt>{{'android_app.os_android_version.title' | translate}}</dt>
-          <dd>
+        </div>
+      </div>
+      <div class="cell" *ngIf="androidDeviceInfo?.app?.version">
+        <div>
```

Comment:
Something got lost in translation... I think you're suggesting removing the `<div>`s?

I think with the current CSS they're needed to fire this line: https://github.com/medic/cht-core/blob/master/webapp/src/css/material.less#L124

This is reusing the same structure as this code: https://github.com/medic/cht-core/blob/master/webapp/src/ts/modules/contacts/contacts-content.component.html#L61

Reply:
Ahh, that's because `contacts-content.component.html#L61` has a label before the `div` and helps to properly align the other elements in relation to that label. Be here we don't have that, so looks like an unnecessary extra structure. 

I think it's safer to use the regular [MatGrid](https://material.angular.io/components/grid-list/overview). The structure is standard, fast to get it to work, less likely to get overridden by accident, and easier to understand. Because this other thing is a custom arrangement from contact-content, it needs more digging before we can do maintenance work.

Reply:
> Ahh, that's because contacts-content.component.html#L61 has a label before the div and helps to properly align the other elements in relation to that label. Be here we don't have that, so looks like an unnecessary extra structure.

Yeah exactly.

MatGrid seems overkill as this is mostly a vertical list not a grid (the data usage part is a grid but I haven't modified that at all). Wouldn't a [list](https://m3.material.io/components/lists/overview) be more fitting?

Reply:
I was thinking the real state suggestion from Nicole, MatGrid makes columns easier to do. The [MatList](https://material.angular.io/components/list/overview) also works

Reply:
MatGrid is no good because the second column isn't responsive so we'd end up with not enough horizontal space on mobile again.

I think each section should be a Card with a List inside.

Reply:
What a nightmare. It turns out List doesn't expand to fit the content, it just truncates with an ellipsis, and it's impossible to overwrite because it fixes the element height to the number of lines it expects. In other words it's not responsive at all.

Cards are pretty but also not responsive, so if we want a grid of cards we're going to have to implement it ourselves anyway.

Reply:
I've gone for cards but ignored responsive design and gone for mobile first instead.